app/javascript/vue/protocol/step.vue

```diff
@@ -24,7 +24,7 @@
             @click="toggleCollapsed">
               <span class="fas fa-caret-right"></span>
           </a>
-          <div class="step-complete-container" :class="{ 'step-element--locked': inRepository || !urls.state_url }">
+          <div v-if="!inRepository" class="step-complete-container" :class="{ 'step-element--locked': inRepository || !urls.state_url }">
```

Comment:
in this case you can remove inRepository from this condition:
`'step-element--locked': inRepository || !urls.state_url`